### DISCUSSION TITLE
Table editor as a Distinct offering

### DISCUSSION THREAD

**Discussion by crossphoton** [supabase/supabase#7192] [Feature Requests]
Hi,
I'm new to Supabase development and not familiar with the work done here but have used the platform.

What I really liked was the Table Editor and wanted to see a distinct project of it where I can pass a Postgres URI and it just works. I'm not sure but guessing if it uses Postgres rest APIs for the purpose so that brings up other issues.
I saw [supabase/grid](https://github.com/supabase/grid) but it is archived.

Lemme know what you think and if any previous discussion has happened already regarding this.

**Comment by steve-chavez** [comment]
>  I'm not sure but guessing if it uses Postgres rest APIs

FYI, it doesn't. The grid creates its own queries.

Also note that you can self-host and run Supabase locally with the [CLI](https://github.com/supabase/cli).

**Reply by crossphoton** [reply]
Thanks! I'll check it out.
